apps/passport-client/components/screens/LoginScreens/CreatePasswordScreen.tsx

```diff
@@ -123,6 +139,8 @@ export function CreatePasswordScreen(): JSX.Element | null {
 
   if (settingPassword) {
     content = <ScreenLoader text="Creating your account..." />;
+  } else if (autoRegister) {
+    content = <ScreenLoader text="Automatically creating your account..." />;
```

Comment:
I'm not sure that the word `Automatically` is worth hiding/showing to the end-user